src/plugins/opensearch_dashboards_react/public/overview_page/overview_page_header/overview_page_header.test.tsx

```diff
@@ -31,95 +31,265 @@
 import React from 'react';
 import { OverviewPageHeader } from './overview_page_header';
 import { shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { useOpenSearchDashboards } from '../../context';
+import { getLogosMock } from '../../../../../core/common/mocks';
 
-jest.mock('../../app_links', () => ({
-  RedirectAppLinks: jest.fn((element: JSX.Element) => element),
-}));
-
-jest.mock('../../context', () => ({
-  useOpenSearchDashboards: jest.fn().mockReturnValue({
-    services: {
-      application: { capabilities: { navLinks: { management: true, dev_tools: true } } },
-      notifications: { toast: { addSuccess: jest.fn() } },
-    },
-  }),
-}));
-
-afterAll(() => jest.clearAllMocks());
+jest.mock('../../context', () => ({ useOpenSearchDashboards: jest.fn() }));
 
 const mockTitle = 'Page Title';
 const addBasePathMock = jest.fn((path: string) => (path ? path : 'path'));
 
+const mockProps = () => ({
```

Comment:
Another one where it's not immediately clear why it's a function call and not a constant object